engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/AvroSerializationSchemaFactory.scala

```diff
@@ -5,41 +5,32 @@ import org.apache.kafka.common.serialization.Serializer
 import pl.touk.nussknacker.engine.kafka.KafkaConfig
 import pl.touk.nussknacker.engine.kafka.serialization.{KafkaKeyValueSerializationSchemaFactoryBase, KafkaSerializationSchemaFactoryBase}
 
-class AvroSerializationSchemaFactory(schemaRegistryClientFactory: SchemaRegistryClientFactory) extends KafkaSerializationSchemaFactoryBase[Any]  {
-
+trait AvroSerializer {
```

Comment:
Why not factory? It creates so it is factory

Reply:
Changed.